README.md

```diff
@@ -80,6 +80,8 @@ Configure the plugin in the `.config/plugins.js/ts` file of your Strapi project.
 | `width`              | `number` <br/> Min: 0                  | The width of the output image in pixels. If only width is specified then the height is calculated with the original aspect ratio.  |
 | `withoutEnlargement` | `boolean`                              | When true, the image will not be enlarged if the input image is already smaller than the required dimensions. Default is `false`.  |
 
+If both width and height are not set, the output will be the same size as the original. This allows for transforming into different formats with the original size.
```

Comment:
Same here. I think it makes more sense to add this sentence to the width and height descriptions themselves to make things clearer, even though in this case it would be doubled. We could also remove the second sentence to shorten the descriptions, because in my opinion this is self-explanatory.